docs/docsite/rst/dev_guide/developing_modules_in_bulk.rst

```diff
@@ -0,0 +1,156 @@
+
+If you need further advice join the `#ansible-devel` IRC channel (details in "Where to get support")
+
+
+We have a "ansibot" helper that comments on GitHub Issues and PRs which should highlight important information.
+
+
+Subsequent PRs
+``````````````
+
+By this point you first PR that defined the module namespace should have been merged. You can take the lessons learned from the first PR and apply it to the rest of the modules.
+
+Raise exactly one PR per module for the remaining modules.
+
+Over the years we've experimented with different sized PRs, some containing one module, some containing five, some even containing many tens of modules in, we've found the following:
+
+* A PR with a single file gets a higher quality review
+* PRs with multiple modules are harder for the creator to ensure all feedback has been applied
+* Lower priority to review. People generally review the easier things first, what would you review first a PR with one file, or five?
+
+FIXME, should we tell people to only raise one at a time, or (say) 5 PRs at once?
+
+
+Finally
+```````
+
+Now that your modules are integrated there are a few bits of housekeeping to be done
+
+**Maintainers**
+Update `Ansibullbot` so it knows who to notify if/when bugs or PRs are raised against your modules
+`MAINTAINERS.txt <https://github.com/ansible/ansibullbot/blob/master/MAINTAINERS.txt>`_.
+
+If there are multiple people that can be notified, please list them. That avoids waiting on a single person who may be unavailable for any reason. Note that in `MAINTAINERS.txt` you can take ownership of an entire directory.
+
+
+**Review Module web docs**
+Review the autogenerated module documentation for each of your modules, found in `Module Docs <http://docs.ansible.com/ansible/modules_by_category.html>`_ to ensure they are correctly formatted. If there are any issues please fix by raising a single PR.
+
+If the module documentation hasn't been put live yet let a member of the Ansible Core Team know in `#ansible-devel` IRC channel.
+
+
+.. seealso::
+
+# FIXME
+   `irc.freenode.net <http://irc.freenode.net>`_
+       #ansible-devel IRC chat channel
+
+
+* FIXME: Fix heading levels (Before you start, (sub, sub), Your first PR
+* FIXME: Review all links
+* FIXME: Review all anchors
+* FIXME: What's a better filename and title for this page?
+* NOTE:  Do we want to guide partners into the standard community work flow? (How can we title/aim this page so it does that)
```

Comment:
Yes, I think all modules partners and otherwise should be the same flow to ensure consistency.  I would parent/child the page flow so all information goes from one page to another.  Targeting bulk submission or single module.

To me this page is more a what makes a module, it's namespace, and best practices/howto.  Then we can go into more detail in bulk submission on one page, and single module submission with in depth practices later.